File: ClothingWarehouseProjectFRONT/src/app/model/size.ts
import {SizeNaming} from "./size-naming";

export class Size{

   id: number
   surcharge: number
   sizeNaming: SizeNaming

  constructor() {
    this.id = 0
    this.surcharge = 0
    this.sizeNaming = new SizeNaming()
  }

}

File: ClothingWarehouseAPI/src/main/java/com/example/clothingwarehouseproject/domain/Size.java
package com.example.clothingwarehouseproject.domain;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import javax.persistence.*;

@Entity
@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
public class Size {

    @Id
    @GeneratedValue(strategy = GenerationType.AUTO)
    private long id;
    @OneToOne(fetch = FetchType.LAZY)
    private SizeNaming sizeNaming;
    private Double surcharge;

}
